src/spin1_beam_model/cst_processing.py

```diff
@@ -73,43 +85,59 @@ def __init__(self, data_files, data_freqs, spatial_sampling, data_uncertainty,
 
         data_file = np.loadtxt(data_files[0], skiprows=2)
 
-        self.Ntheta = int(181./spatial_sampling)
-        self.Nphi = int(360./spatial_sampling)
+        self.Ntheta = int(181.0 / spatial_sampling)
```

Comment:
Probably not the right place to ask this (and @zacharymartinot would be the one to know), but why is this 181 instead of 180? Some kind of periodicity issue?